### PR TITLE
Site header

### PR DESCRIPTION
**Pull request by tmccoy529**
SiteHeader #78 and Logotype #70 have been added to this PR. A placeholder for Beta/Search has been added as well but this is for testing sizing for now and is a place holder. Until we get the other items added to the header the logo will not pull left on mobile as required as it is difficult to see what needs to be adjusted until its closer to completion.

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -11,3 +11,37 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.header {
```
Comment (theilman7) [inline src/index.css:15]: I would recommend using the global variables contained in the styles/variables/border.scss file in addition to the global brand color variables used in styles/variables/colors.scss. This should this also be saved as a .scss file and imported in to the appropriate section of dotgov.scss.

File: src/index.css [hunk 2]
```diff
@@ -11,3 +11,37 @@ code {
...
+  text-transform: uppercase;
+}
+
+.brand span:first-of-type {
+  font-weight: 700;
+  font-size: 18px;
+  color: #000000;
+}
+
+.brand span:last-of-type{
```
Comment (theilman7) [inline src/index.css]: Recommend organizing properties in alphabetical order for consistency in the source.

File: src/index.css [hunk 3]
```diff
@@ -11,3 +11,37 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
```
Comment (theilman7) [inline src/index.css]: Recommend assigning this a variable from the /styles/variables/fonts.scss file.

File: src/index.css (hunk 2 above)
Comment (tmccoy529) [reply src/index.css]: Will do

File: src/index.css (hunk 1 above)
Comment (tmccoy529) [reply src/index.css:15]: So the body and code css isnt something that I created (I think these are react defaults) and Im actually not sure what they are and will most likely be ignored/removed if this project starts using SASS. As far as incorporating these into dotgov.scss I think that is a good idea but Marty wasn't sure how he wanted to handle the css for this project. Either importing from dotgov-components or having its own separate css files. Depending on that decision then we can move accordingly.

**Comment by martypowell** [comment]
@theilman7 this is eventually going to be abandoned per a conversation between @tmccoy529 and I. A new pr will be created against dotgov-components, and we will address any concerns there.

**Pull request closed without merging**